courses/free-courses-en.md

```diff
@@ -989,6 +989,8 @@
 
 ### Networking
 
+* [Computer Networking](https://www.youtube.com/watch?v=IPvYjXCsTg8) - Kunal Kushwaha
```

Comment:
```suggestion
* [Computer Networking Full Course](https://www.youtube.com/watch?v=IPvYjXCsTg8) - Kunal Kushwaha
```
Use the same name given by the resource

Reply:
Does it really matter? Either may I'm committing your suggestion. And, please suggest all the suggestion in one go

Reply:
Hmm yeah, it's in the contribution guidelines. Many resources have license to share but with proper attribution, so we prefer to have the same name that is in the resource